.circleci/config.yml

```diff
@@ -69,28 +95,21 @@ jobs:
         type: enum
         enum: ['x64', 'arm64']
     executor: << parameters.executor >>
+    parallelism: 2
     steps:
       - install
       - when:
           condition:
             equal: [node/windows, << parameters.executor >>]
           steps:
-            - node/install-packages
             - run:
-                name: setup Windows and run slow tests
-                shell: powershell.exe -ExecutionPolicy Bypass
+                name: Windows Setup
+                shell: bash
                 command: |
-                  Write-Output "Installing WiX Toolset"
                   choco install --no-progress -y wixtoolset
-                  Write-Output "Reloading PATH"
-                  $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
-                  [Environment]::SetEnvironmentVariable("PATH", $env:Path + "C:\Program Files (x86)\WiX Toolset v3.11\bin;", "Machine")
-                  $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
-                  npm config set node-gyp "$CIRCLE_WORKING_DIRECTORY\node_modules\node-gyp\bin\node-gyp.js"
-
-                  yarn lint
-                  yarn build
-                  yarn test:slow
+                  echo 'export PATH=$PATH:"/C/Program Files (x86)/WiX Toolset v3.11/bin"' >> "$BASH_ENV"
```

Comment:
I don't suppose we could cache the entire Wix installation?

Reply:
I'm not sure what `choco install wixtoolset` does to the system aside from put the files in the right place.

I could give it a whirl in a separate PR to see if there are any real gains to be made, though?

Reply:
Fair enough